app/api/ada/index.js

```diff
@@ -739,6 +740,43 @@ export default class AdaApi {
     }
   }
 
+  async createSendAllTransaction(
+    request: CreateTransactionRequest
+  ): Promise<CreateTransactionResponse> {
+    Logger.debug('AdaApi::createSendAllTransaction called');
+    const { receiver, password } = request;
+    try {
+      const unsignedTx = await sendAllUnsignedTxFromUtxo(
```

Comment:
I feel kind of weird about the code duplication when only this line is diff

Reply:
Actually, the line 751 have to be modified to get all the UTXO of the wallet

Reply:
You can unify this point in the API layer. `newAdaUnsignedTx` gets the UTXO from a lambda and `sendAllUnsignedTxFromUtxo` takes them passed in as an argument but you could standardize it so they just both take the UTXO as an argument.